README.md

```diff
@@ -132,5 +132,14 @@ Here is the full set of keywords goldentests looks for in the file:
 - `expected exit status: [i32]`: If specified, goldentests will issue an error if the exit status differs
   to what is expected. Defaults to `None` (exit status is ignored by default).
 
+
+
 You can even configure the specific keywords used if you want. For any further information,
 check out goldentest's documentation [here](https://docs.rs/goldentests).
+
+### Update all tests with current output
+Set the `overwrite_tests` field in the config if you're running using cargo.
+The standalone executable checks for the flag `--overwrite` or the environment variable `GOLDENTEST_OVERWRITE`.
+
+`overwrite tests`: If you want to update all your tests to match the current output, the `overwrite_tests` flag is for you.
```

Comment:
This is worded a bit oddly, lets remove this last sentence and explain what the overwrite_tests flag does before explaining how to enable it

Reply:
I really struggle with my technical writing. I reformulated it, but I'm not entirely happy with it. I'm very open to criticism here and thanks for the pointers.